### ISSUE TITLE
Keep getting "can't render header" on twitter auth

### ISSUE THREAD

**Issue by anton-107** [ciaranj/connect-auth#40]
It seems, that /auth/twitter_callback can't write headers because it depends on Connect.session(), which write headers itself.

Am i right?

**Comment by sf-old** [comment]
I get the same issue (with other services as well such as Facebook).

```
{ stack: [Getter/Setter],
  arguments: undefined,
  type: undefined,
  message: 'Can\'t render headers after they are sent to the client.' }
Error: Can't render headers after they are sent to the client.
    at ServerResponse._renderHeaders (http.js:571:11)
    at ServerResponse.writeHead (http.js:804:20)
    at /Users/ifoundaway/we/app.js:70:8
    at /usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/index.js:95:41
    at next (/usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/strategyExecutor.js:45:39)
    at [object Object].pass (/usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/authExecutionScope.js:18:3)
    at [object Object].halt (/usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/authExecutionScope.js:15:8)
    at [object Object].redirect (/usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/authExecutionScope.js:23:8)
    at /usr/local/lib/node/.npm/connect-auth/0.2.2/package/lib/auth.strategies/twitter.js:65:16
    at /usr/local/lib/node/.npm/oauth/0.9.0/package/lib/oauth.js:412:7
```

**Comment by dustyjewett** [comment]
Still seeing this with Facebook and Twitter.  Are there any workarounds?

**Comment by ciaranj** [comment]
Implemented in 0.3.0
